reviews/layout/helpers.py

```diff
@@ -30,28 +30,36 @@ def render_pull_request_table(
     pull_requests = sorted(pull_requests, key=lambda x: x.updated_at, reverse=True)
 
     for pr in pull_requests:
-        approved = ""
-        if pr.approved == "APPROVED":
-            approved = "[green]Approved"
-        elif pr.approved == "CHANGES_REQUESTED":
-            approved = "[yellow]Changes Requested"
-
-        approved_by_others = "[green]Ready" if pr.approved_by_others else ""
+        # Formatting PR Title Column
+        title_colour = "[white]"
+        if "Security" in pr.title:
+            title_colour = "[yellow]"
 
+        # Formatting Activity Column
         updated_at = humanize.naturaltime(pr.updated_at)
-
         colour = ""
         if (datetime.now() - pr.updated_at).days >= 7:
             colour = "[red]"
         elif (datetime.now() - pr.updated_at).days >= 1:
             colour = "[yellow]"
         updated_at = f"{colour}{updated_at}"
 
+        # Formatting Approved Column
+        approved = ""
+        if pr.approved == "APPROVED":
+            approved = "[green]Approved"
+        elif pr.approved == "CHANGES_REQUESTED":
+            approved = "[yellow]Changes Requested"
```

Comment:
whilst we are in the area, we should update this to `red` also to match against GitHub colours for approved/changes requested

```suggestion
            approved = "[red]Changes Requested"
```